#Details of service data types
string inputData
---
string outputData